### ISSUE TITLE
Deactivate the import of indicators from HDRSTATS, since those data are no longer available.  

### ISSUE THREAD

**Issue by cjhendrix** [OCHA-DAP/Data-Team#19]
This will avoid the creation of collections on CKAN that fail when the resource is downloaded.

If you can do this by Monday COB, it will be included in the Beta0.1 release on Production.

**Comment by cjhendrix** [comment]
Two alternatives to solving this problem: 

1) Alternatively, we could dig into our our old copies of the SW data to find this and load it to the database.  It wouldn't update, but at least the historic data would be there.

2) Dragon says they have searched to see if this data is out there somewhere, but haven't found it.  You could look for an alternative source, and/or have another look around the HDI website to see if they have recently put it back up.

I'd suggest we do 1 while we are working on 2.

**Comment by luiscape** [comment]
Sorry, we didn't read this notice. We'll be watching this issue tracker more closely now. Use mentions (@) to get our attention.

**Comment by luiscape** [comment]
We are closing this issue -- couldn't understand if the issue still persists.

**Comment by cjhendrix** [comment]
Actually, it is still an issue, but it can't be resolved by deactivating the import in CPS Metadata Gdoc (since that is frozen).  What we need to do now is delete all the collections for which HDRStats is the source.    These collections exist, but the downloaded XLSX have no data because the data has been removed and therefore the scrapers are empty for those indicators. 

I've deleted _HDR:68606 and PSE110 already, but that leaves another 10 or so.
